src/Sulu/Bundle/LocationBundle/Resources/js/index.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import {fieldRegistry} from 'sulu-admin-bundle/containers';
+import L from 'leaflet';
+import leafletMarkerIcon from 'leaflet/dist/images/marker-icon.png';
+import leafletMarkerShadow from 'leaflet/dist/images/marker-shadow.png';
+import {Location} from './containers/Form';
+
+// leaflet requires that its stylesheet is embedded somewhere into the build
+// eslint-disable-next-line
```

Comment:
What rule is disabled here? Add the rule name after the comment, then the other rules still apply.